qemu/tests/driver_in_use.py

```diff
@@ -33,6 +33,8 @@ def check_bg_running(target_process):
         list_cmd = params.get("list_cmd", "wmic process get name")
```

Comment:
This command should be windows only.

Reply:
Thanks, have added judgement for linux and windows